AlphaWallet/Tokens/ViewControllers/TokensViewController.swift

```diff
@@ -368,6 +368,7 @@ extension TokensViewController: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        updateTableViewBackground()
```

Comment:
- [ ] It works, but seems to be unsafe because modifying the `UITableView` instance (although it's just the view) during one of the data source callbacks might become dangerous in the future.

There's now a `TokensViewController.reloadTableData()`. Maybe we can move the call into it?

```
private func reloadTableData() {
    tableView.reloadData()
    updateTableViewBackground()
}
```

It's possible we forget to call `reloadTableData()` and call `tableView.reloadData()` directly, but at least it's just a cosmetic bug.

Reply:
This done.